src/biokbase/narrative/tests/test_jobcomm.py

```diff
@@ -131,41 +128,27 @@ def get_app_data(*args):
     return {"info": {"name": APP_NAME}}
 
 
-def get_retried_job(job_id):
-    return {
-        "jobState": {
-            "job_id": job_id[::-1],
-            "status": "unmocked",
-            "job_output": {},
-            "batch_id": None,
-            "cell_id": None,
-            "run_id": None,
-            "child_jobs": [],
-        },
-        "outputWidgetInfo": None,
-    }
-
-
 class JobCommTestCase(unittest.TestCase):
     maxDiff = None
 
     @classmethod
     @mock.patch("biokbase.narrative.jobs.jobcomm.Comm", MockComm)
     @mock.patch(CLIENTS, get_mock_client)
     def setUpClass(cls):
-        cls.jm = biokbase.narrative.jobs.jobmanager.JobManager()
         config = ConfigTests()
         os.environ["KB_WORKSPACE_ID"] = config.get("jobs", "job_test_wsname")
-
-        cls.jc = biokbase.narrative.jobs.jobcomm.JobComm()
+        cls.jm = JobManager()
+        cls.jc = JobComm()
         cls.jc._comm = MockComm()
+        cls.job_states = get_test_job_states()
+        cls.retry_job = get_test_job_retries()
+        cls.job_info = get_test_job_infos()
```

Comment:
pre-generate all this data so the tests can just take the appropriate slice of the dictionary to get the expected results